website/docs/language/providers/index.mdx

```diff
@@ -93,6 +93,26 @@ installing providers.
 
 > **Hands-on:** Try the [Lock and Upgrade Provider Versions](/terraform/tutorials/configuration-language/provider-versioning?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial.
 
+### Private Providers
+
+If you have a provider that isn't available in a Hashicorp-hosted registry, such
```

Comment:
Making one bigger suggestion for readability. I think this should be two separate sentences to really drive the point home:
```suggestion
If you are using a provider that is not in a Hashicorp-hosted registry, you may need to attach additional credentials to your requests to external registries. You do not need these credentials if your provider is the Terraform public registry or the HCP Terraform private registry.
```